ksqldb-tools/src/main/java/io/confluent/ksql/tools/migrations/util/CommandParser.java

```diff
@@ -263,4 +293,37 @@ public String getSourceName() {
       super(command);
     }
   }
+
+  /*
+   * Represents set/unset property commands.
+   * */
+  public static class SqlPropertyCommand extends SqlCommand {
+    private final boolean set;
+    private final String property;
+    private final String value;
+
+    SqlPropertyCommand(
+        final String command,
+        final boolean set,
+        final String property,
+        final String value
+    ) {
+      super(command);
+      this.set = set;
+      this.property = property;
+      this.value = value;
```

Comment:
nit: convert to Optional to clarify that this may not be present. Then we can also add null checks on `property` and `value` for extra clarity.